Magento_Cms/web/js/custom-country-form.js

```diff
@@ -0,0 +1,66 @@
+define(['jquery', 'mage/url', 'jquery-ui-modules/widget', 'jquery/validate'], (
```

Comment:
as I said in lectures don't use any scripts related to the `jquery` folder. magento 2 has a lot of custom widgets and scripts as wrappers over the jquery script.
use this instead
```js
'mage/validation'
```
or just
```js
'validation'
```